File: 598. Range Addition II/Python/Solution.py
class Solution:
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        result_m = m
        result_n = n
        for op in ops:
            result_m = min(result_m, op[0])
            result_n = min(result_n, op[1])
        return result_m * result_n

File: 598. Range Addition II/C++/Solution.cpp
class Solution {
public:
    int maxCount(int m, int n, vector<vector<int>> &ops) {
        int resultM = m;
        int resultN = n;
        for (const vector<int> &op : ops) {
            resultM = min(resultM, op[0]);
            resultN = min(resultN, op[1]);
        }
        return resultM * resultN;
    }
};